src/models/hooks/goalFieldResponse.js

```diff
@@ -68,6 +68,11 @@ const syncActivityReportGoalFieldResponses = async (sequelize, instance, options
           },
         },
       );
+
+      // Get a list of activity report ids from idsToUpdate.
+      const activityReportIds = idsToUpdate.map((item) => item.activityReportGoal.activityReportId);
+      // We need to update the AR createdAt so we don't pull from outdated local storage.
+      await sequelize.query(`UPDATE "ActivityReports" SET "updatedAt" = '${new Date().toISOString()}' WHERE id IN (${activityReportIds.join(',')})`);
```

Comment:
I think we do need to check the length of "activityReportIds," it looks like at least for test purposes. (Bright side: the path is already tested)

![Screenshot 2023-12-19 at 4 42 17 PM](https://github.com/HHS/Head-Start-TTADP/assets/3288586/8c071d24-ac8d-421a-9b3d-0f2ad5153cc2)

Reply:
Yea I already added that funny that I brought that up when we looked at it lol